### ISSUE TITLE
Isle of Man flag

### ISSUE THREAD

**Issue by AndalfTheBeard** [Layerful/sassy-flags#2]
Hey, any chance of getting a sassy flag for the Isle of Man using the IM country code?

**Comment by dthms** [comment]
Yeah no problem at all, I will sort that out over the next few days!
